release: qase-playwright 2.0.2 (#612)
qase-tms/qase-javascript: qase-playwright/changelog.md

# playwright-qase-reporter@2.0.2

## What's new

> Added new annotation for test case ignoring `qase.ignore()`.
> This annotation will mark the test as ignored in the Qase.io test run.

Reviewer: it's important to note that test will not be ignored by Playwright itself, just won't be reported 
```suggestion
Added new annotation `qase.ignore()`.
Tests marked with it will run as usual but won't appear in the Qase report.
```

Author: fixed
